domain/asset_maintenance.go

```diff
@@ -0,0 +1,15 @@
+package domain
+
+import (
+	"time"
+)
+
+type UUID [16]byte
```

Comment:
type UUID is declared in multiple files.
For UUID generation, you can use github.com/google/uuid package